h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -876,7 +876,7 @@ private void setStoppingCriteria(Model.Parameters parms, Model.Parameters defaul
       parms._stopping_metric = buildSpec.build_control.stopping_criteria.stopping_metric();
 
     if (parms._stopping_metric == StoppingMetric.AUTO) {
-      String sort_metric = getSortMetric();
+      SortMetric sort_metric = getSortMetric();
```

Comment:
also needs to be changed below: `sort_metric = SortMetric.auc`

Reply:
👍